### PR TITLE
fix/user_attrbutes_null

### PR DESCRIPTION
**Pull request by AndriyAndriyovuch**
## Code reviewers

- [ ] @loqimean 

## Summary of issue

Change user's first_name and last_name null to false

## Summary of change

- changed null:false in db
- added columns in admin's users table
- fix test

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: db/migrate/20231019194527_change_user_attributes_null.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class ChangeUserAttributesNull < ActiveRecord::Migration[6.1]
+  def up
+    User.where(first_name: nil).update_all(first_name: "Firstname") # rubocop:disable Rails/SkipsModelValidations
+    User.where(last_name: nil).update_all(last_name: "Lastname") # rubocop:disable Rails/SkipsModelValidations
```
Comment (loqimean) [inline db/migrate/20231019194527_change_user_attributes_null.rb:4]: рефактори)

File: app/controllers/account/users_controller.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@ def index
       format.html
       format.csv do
         UserReportJob.perform_later
-        send_data UsersCsvGenerator.call(@users, fields: ["email", "last_sign_in_at"])
+        send_data UsersCsvGenerator.call(@users, fields: ["email", "first_name", "last_name", "last_sign_in_at"])
```
Comment (loqimean) [inline app/controllers/account/users_controller.rb:20]: такс, чому каверидж падає?)

File: spec/requests/account/users_spec.rb [hunk 3]
```diff
@@ -5,16 +5,28 @@
...
+    it "returns a successful html response" do
       get account_users_path
 
       expect(response).to be_successful
       expect(response).to render_template(:index)
       expect(response.body).to include(I18n.t("account.users.index.main_header"))
     end
+
+    it "returns a successful csv response" do
+      get account_users_path(format: "csv")
```
Comment (loqimean) [inline spec/requests/account/users_spec.rb:20]: це в біфор

File: spec/requests/account/users_spec.rb [hunk 4]
```diff
@@ -5,16 +5,28 @@
...
+
+      csv_content = response.instance_variable_get(:@stream).instance_variable_get(:@buf).join
```
Comment (loqimean) [inline spec/requests/account/users_spec.rb:22]: а це в звичайний let, і тоді не тре буде змінної в тесті

**Review by loqimean — APPROVED** [review]

**Comment by NatLabyk** [comment]
Done

**Pull request merged**